plugins/flytekit-flyin/flytekitplugins/flyin/vscode_lib/decorator.py

```diff
@@ -171,6 +173,61 @@ def download_vscode(vscode_config: VscodeConfig):
         execute_command(f"code-server --install-extension {p}")
 
 
+def prepare_interactive_python(task_function):
+    """
+    1. Copy the user's Python file to the working directory. This ensures that the inputs.pb can be loaded, even when the user switches the task interface.
```

Comment:
```suggestion
    1. Copy the original task file to the context working directory. This ensures that the `inputs.pb` can be loaded because loading requires the original task interface. By doing so, even if users change the task interface in the user code, we can use the copied task file to load the inputs as native python objects.
```